src/client/sandbox/event/focus-blur.js

```diff
@@ -233,6 +233,9 @@ export default class FocusBlurSandbox extends SandboxBase {
     }
 
     focus (el, callback, silent, forMouseEvent, isNativeFocus, preventScrolling) {
+        if (el.disabled)
+            return null;
+
```

Comment:
lgtm.

But before merge please recheck whether we need the `this.shouldDisableOuterFocusHandlers` property check below. I can't find any place where we set it to `true`

Reply:
Remove the flag `this.shouldDisableOuterFocusHandlers` and whole condition
```js
if (this.shouldDisableOuterFocusHandlers && !domUtils.isShadowUIElement(el))
            return null;
```
It never used in `testcafe`, `testcafe-hammerhead`, `testcafe-studio` and `testcafe-legacy-api` repositories.

Reply:
Done.